CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+- #685 - Add android libc version
```

Comment:
it's fine to just add this to the previous line, no need for a separate entry

Reply:
or, we can skip modifying changelog entirely

Reply:
I removed the commit.
Now changelog check is failed. Is it OK?

Reply:
Yes 👍